manifests/mirror.pp

```diff
@@ -29,6 +29,6 @@
     architectures => $architectures,
     components    => $components,
     with_sources  => $with_sources,
-    with_udebs    => $with_udebs
+    with_udebs    => $with_udebs,
```

Comment:
This change doesn't really make sense to me (and it's not coherent with the rest of the manifests). Why do you want to change that here? What's the lint error you're trying to solve?